### ISSUE TITLE
Running ICA on smoothed data

### ISSUE THREAD

**Issue by tsalo** [ME-ICA/tedana#565]
### Summary
I have a quick question relevant to using MELODIC with tedana within fMRIPrep. Do folks think it would be a problem to run the ICA on smoothed optimally combined data just to get the component time series, and then run the classification procedure on the unsmoothed data like we normally would? Essentially, users would provide `--mix` using a mixing matrix from MELODIC run after SUSAN, but would still provide the pre-SUSAN data as `-d`.

### Additional Detail
If this is feasible, then it would make it a lot easier to integrate tedana's denoising into fMRIPrep in parallel with AROMA, rather than having to replace it. For a more complete picture, the steps in fMRIPrep would be:

1. Run t2smap to get optimally-combined data from unsmoothed multi-echo data.
     - The T2* map could also be retained for the tedana run later.
2. Smooth data with SUSAN (already part of AROMA workflow within fMRIPrep).
3. Run MELODIC on smoothed data (part of fMRIPrep AROMA workflow).
4. Retain mixing matrix and decomposition files from ICA.
5. (For tedana workflow) Run tedana on unsmoothed multi-echo data, providing `--mmix` from MELODIC.
    - This updates the MELODIC decomposition file with classifications.
6. (For AROMA workflow) Run ICA-AROMA on smoothed optimally-combined data, using files from MELODIC.
    - This updates the MELODIC decomposition file with classifications.

**Comment by CesarCaballeroGaudes** [comment]
The order of the ICA decomposition with smoothed data will be lower than with unsmoothed data, and the time series will also be different. Although it is not ideal, it would be a pragmatic approach since the IC time series (--mmix) are used as a model for linear regression within tedana. 
I've not worked with fMRI prep. Why is SUSAN applied prior to MELODIC and ICA-AROMA in fMRIprep? Is it to reduce the order of the decomposition and ease the classification between components?

**Comment by dowdlelt** [comment]
This would produce maps from the unsmoothed data using the smoothed data component timeseries, right?

**Comment by CesarCaballeroGaudes** [comment]
Yes

**Comment by tsalo** [comment]
> Why is SUSAN applied prior to MELODIC and ICA-AROMA in fMRIprep?

The [ICA-AROMA manual](https://github.com/maartenmennes/ICA-AROMA/blob/e8d7a5891563aff7ff71375078314c417d32d808/Manual.pdf) says that it must be run that way. Per the manual:

> Note that MELODIC should have been run on fMRI data prior to temporal filtering and after spatial smoothing.

My assumption is that this is required due to the spatial nature of some of AROMA's features (e.g., edge and CSF weights), although I will probably have to re-read the original paper to be sure.

**Comment by dowdlelt** [comment]
I feel like this might actually work quite well - the components from the smoothed data would be the structured ones, so to speak (because of the reduction in the model order/suppression of noise from smoothing). This makes me think that reproducing them on the unsmoothed data would be a reasonable strategy, and would lead to reasonable rho/kappa values. 

Smoothing data prior to tedana was always hit or miss for me - but it makes sense because the spatial classifications step had to contend with a very different type of map compared to the original implementation. 

I'm pro this idea. At the very least, it may work well. I've often wondered, on my datasets that failed to converge, if I should be hacky and take something like this approach with my data.

**Comment by CesarCaballeroGaudes** [comment]
Well, my opinion is that ICA-AROMA might work better rather than it is 'required'. I have also used on unsmoothed data and it also worked nicely. Of course, the distinction of the components is more difficult and the number of components is larger without smoothing.
Is it possible to run MELODIC without SUSAN in fMRIprep?

**Comment by tsalo** [comment]
At the moment it isn't, but I'm hoping to restructure the workflow to accommodate tedana soon. It should be easy enough to drop SUSAN at that point- assuming it doesn't ruin the AROMA classifications.

**Comment by CesarCaballeroGaudes** [comment]
Yes, I guess the goal would be to add the AROMA features along with the tedana ones, isn't it? That would be very helpful indeed.  The lack of smoothing should not ruin the AROMA classifications dramatically. Keep us posted. As you know, it is something we always wanted to do with @smoia

**Comment by tsalo** [comment]
I'll move forward with issues in fMRIPrep once interested folks over there are able to respond (so far there haven't been many replies to my comments). The idea of applying tedana and AROMA to the same decomposition has already gotten some upvotes, so the main thing now is figuring out _how_ to do it. To build off of this conversation, I will propose first that SUSAN be dropped (i.e., run MELODIC and AROMA on unsmoothed data). If that is a nonstarter according to the AROMA experts, then I will try to implement what is proposed above (i.e., run MELODIC on smooth data, then run tedana on unsmoothed data).

Thank you both for your insights!

**Comment by CesarCaballeroGaudes** [comment]
What's the issue number in fMRIprep?

**Comment by tsalo** [comment]
The relevant issues are https://github.com/poldracklab/fmriprep/issues/1784 and https://github.com/poldracklab/fmriprep/issues/1010.
